.travis.yml

```diff
@@ -14,53 +15,89 @@ env:
     - BASEPATH: "$TRAVIS_BUILD_DIR"
     - ROOTSYS: "$BASEPATH/root"
 
+addons:
+  homebrew:
+    packages:
+      - boost
+      - python3
+      - pip3
+  apt:
+    update: true
+    packages:
+      - libboost-all-dev
+      - python3
+      - python3-pip
+      - python3-venv
+      - pandoc
+
+before_install:
+  - cd $BASEPATH
+  - python3 -m venv venv
+  - source ./venv/bin/activate
+
+install:
+  - python3 -m pip install scikit-build
+  - python3 -m pip install pytest==5.4.1
+  - python3 -m pip install pytest-cov
+  - python3 -m pip install scipy
```

Comment:
why you do this manually again?

Reply:
Not installing other dev tools (such as `pylint`) saves around 30s

Reply:
yeah but then we need a `requirements_dev` and `requirements_prod`

Reply:
Hm good point. So would `pytest` be in dev and autopep in prod?

Reply:
I would put everything in dev, and only the run requirements into prod. Not sure if we should also define an extra test requirements file, so that prod would only include the bare essentials.

Reply:
> so that prod would only include the bare essentials

But that's what the `requirements.txt` file is for. This file is even used by e.g. GitHub to list the dependencies of the project.

Reply:
yeah. I mean something like this https://stackoverflow.com/questions/17803829/how-to-customize-a-requirements-txt-for-multiple-environments

Reply:
Ah yeah that would be useful for instance in the data module https://github.com/ComPWA/pycompwa/pull/78 where uproot is a dependency (it is currently losely a dependency in the rootdataplot module, but that should be abandoned anyway). With such nested requirements, the modules would be come a more independent as well.

Reply:
Ok, so this requirements issue will be tackled in a separate PR

Reply:
Yeah sorry forgot to mention: https://github.com/ComPWA/pycompwa/issues/81